state/cloudflare/workerskv/metadata.yaml

```diff
@@ -0,0 +1,58 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: state
+name: cloudflare.workerskv
+version: v1
+status: beta
+title: "Cloudflare Workers KV"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-state-stores/setup-cloudflare-workerskv/
+capabilities:
+  - crud
+authenticationProfiles:
+  - title: "Ed25519 key pair"
```

Comment:
We use "authenticationProfiles" for things like credentials for cloud services. In this case, the Ed25519 key is used for Dapr to authenticate itself with the Worker we deploy, so it's a different thing that should go outside of the auth profile.